test/ui-2/test/clear_step.test.js

```diff
@@ -0,0 +1,71 @@
+const timeout = process.env.SLOWMO ? 30000 : 10000;
+
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+
+describe('Clear Steps', () => {
+  test('Resize Module is added', async () => {
+    
+    await page.waitForSelector('.step');
+    
+    const Length = await page.evaluate(() => document.querySelectorAll('.step').length);
```

Comment:
Could you please add some line breaks in the below code. It is really difficult to read.